convert.py

```diff
@@ -27,5 +49,115 @@
 
         codelist_items.append(codelist_item)
 
+indent(template.getroot(), 0, 4)
 template.write('xml/FileFormat.xml', pretty_print=True)
 
+
+
+"""
+
+ISO Country Alpha 2
+
+"""
+
+XML_LANG = '{http://www.w3.org/XML/1998/namespace}lang'
+
+template = ET.parse('templates/Country.xml', ET.XMLParser(remove_blank_text=True))
+codelist_items = template.find('codelist-items')
+
+def add_code(code_text, country, withdrawn):
+    codelist_item = ET.Element('codelist-item')
+
+    if withdrawn:
+        codelist_item.attrib['withdrawn'] = withdrawn
```

Comment:
The [chosen solution](https://github.com/IATI/IATI-Codelists/issues/89) ended up [adding](https://github.com/IATI/IATI-Codelists/commit/08dcf4c96f56ba504939b435a5521b06e0fdef73) a trio of attributes - `status`, `activation-date` and `withdrawal-date`. As such, this script and the generated Codelists should use that method of identifying withdrawn codes.